src/A1_csv_to_json.py

```diff
@@ -359,10 +360,112 @@ def create_json_from_csv(input_directory, filename, parameters, storage=False):
             "No %s" % filename + " assets are added because all "
             "columns of the csv file are empty."
         )
-
-    for column in df:
+    df_copy = df
```

Comment:
```suggestion
    df_copy = df.copy()
```
Otherwise it does not make an actual copy